Reply/app/src/main/java/com/example/reply/MainActivity.kt

```diff
@@ -10,13 +10,13 @@ import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
-import com.example.reply.ui.theme.ReplyTheme
+import com.example.reply.ui.theme.ReplyM3Theme
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
-            ReplyTheme {
+            ReplyM3Theme {
```

Comment:
I think we can keep this as `ReplyTheme` (otherwise the naming for other things should probably also change eg. `replyM3DarkColorScheme` which would be a bit messy IMO)

Reply:
Agreed 👍🏽